code/modules/power/hydroelectric.dm

```diff
@@ -0,0 +1,243 @@
+//Hydroelectric power by Sieghardt Meldurson. Please message me in case of issues.
+//The Hydro Turbine itself
+#define HYDRO_MAX_DIST 100
+#define DEBRISMALFUNCTION 1
+#define STALLMAFLFUNCTION 2
+#define NOMALFUNCTION 0
+/obj/machinery/power/hydroelectric
+	name = "hydroelectric turbine"
+	desc = "A hydroelectric generator able to store water and then release it to generate electricity. This one seems to be important to the structural integrity around and is attached directly to the dam's frame."
+	icon = 'icons/obj/machines/thermoelectric.dmi'
+	icon_state = "circ-unassembled"
+	density = 1
+	anchored = 1
+	use_power = NO_POWER_USE
+	idle_power_usage = 0
+	active_power_usage = 0
+	health = 10
```

Comment:
Why is the hp so low? I can imagine theirs cases were an engi hits the turbine by misstake with their crowbar and kills it or something... Maybe 300?